spec.html

```diff
@@ -339,6 +339,17 @@ <h1>Cyclic Module Records</h1>
             A List of all the |ModuleSpecifier| strings used by the module represented by this record to request the importation of a module. The List is source code occurrence ordered.
           </td>
         </tr>
+        <tr>
+          <td>
+            <ins>[[CycleRoot]]</ins>
+          </td>
+          <td>
+            <ins>Cyclic Module Record | *undefined*</ins>
```

Comment:
The new editorial convention for fields that hold values that are never supposed to escape to user code is to use `~empty~` instead of undefined.

Reply:
Currently all fields on cyclic module records use the `*undefined*` convention, so it seems like this is a refactoring that is yet to take place for the modules section in ecma262 as far as I can tell? I can add this, but it would be an inconsistency surely? Or am I misunderstanding the definition?

Reply:
Ah.. that is a good point. This field is either going to be empty (uninitialized) or we will have a value here. But it is also true of other fields in module records. 

Maybe we can do this as a separate patch, this would be a good clarification to have. (I can volunteer to do that)

Reply:
Yeah, this editorial issue seems good to handle in a separate PR.

Reply:
Ok will mark this as resolved here to instead land in https://github.com/tc39/proposal-top-level-await/pull/165.